src/utilities.js

```diff
@@ -0,0 +1,113 @@
+import * as Constants from './constants';
+
+export function addLotToPortfolio(ticker) {
+  let portfolio = JSON.parse(window.localStorage.getItem('portfolio'));
+  if (!portfolio) {
+    window.localStorage.setItem(
+      'portfolio',
+      JSON.stringify({
+        name: 'To The Moon',
+        lots: [],
+      })
+    );
+    portfolio = JSON.parse(window.localStorage.getItem('portfolio'));
+  }
+  portfolio.lots.push({
+    symbol: ticker.toUpperCase(),
+    buyShares: 0,
+    buyDate: null,
+    broker: null,
+    buyPrice: null,
+    sellShares: 0,
+    sellDate: null,
+    sellPrice: null,
+  });
+  window.localStorage.setItem('portfolio', JSON.stringify(portfolio));
+}
+
+// TODO: need to convert this to sum values
+export const getDataForChart = (ticker) => {
+  const pricesJson = Constants.API_PRICES[ticker];
+  const xAxisLabels = [];
+  const yAxisLabels = [];
+  for (let i = 0; i < pricesJson.length; i++) {
+    xAxisLabels.push(pricesJson[i].date.split('T')[0]);
+    yAxisLabels.push(pricesJson[i].close);
+  }
+
+  const metaData = Constants.API_META[ticker];
+  const name = metaData['name'];
+
+  return { xAxisLabels, yAxisLabels, name };
+};
+
+export function formatDate(date) {
```

Comment:
I would call this "formatDateISO" or similar (referring to ISO 8601).

Can't you use Date.toIOSString()? (Note: need to worry about time zones)

Reply:
like you said, this will convert the Date object to GMT, which can flip it to the "wrong" day...

Reply:
Might be good to put a comment about why you can't use Date.toISOString()
[JavaScript time functions kinda suck, it seems, and Intl.DateTimeFormat doesn't have an "ISO-8601" locale]